jobs/nicheny-django-developer.html

```diff
@@ -0,0 +1,40 @@
+---
+title: Freelance Django Developer
+company: Niche
+location: New York, NY
+contract: contract
+contact:
+    name: Mark W
+    email: contact@nicheny.com
+created: !!timestamp '2017-03-07' 
+tags:
+  - python
+  - sql
+  - django
+  - freelance
+  - remote
+  - telecommute
+---
+
+# Overview
+
+Startup company seeks freelance Python/Django developer for putting finishing touches on our product.
+We are practically in our eleventh hour, and need additional help to get ready and appropriately scalable for primetime.
+Ongoing maintenance will be needed as well.
```

Comment:
I think it would be helpful to candidates if you gave more of an idea of the requirements. I guess you need an "in at the deep-end" kind of person, right? But do you need anything more specific to get things releasable?